### DISCUSSION TITLE
Option/command line to use another GPU card

### DISCUSSION THREAD

**Discussion by Fhalo48** [hlky/stable-diffusion-webui#1156] [Feature suggestions]
Hi

Where to go to set the software to use another GPU card or CPU? 

Thanlks

**Comment by Rolli700** [comment]
Hello and at first: Thank you for the hard n good work.
Id like to utilize a 2nd cuda Card (GPU:1). Can two cards be combined or will i need Start a whole 2nd webui instance somehow?

Thanks for hints and knowledge.

**Comment by Telecino333** [comment]
Any method suggested to change to gpu 1 gave me the error, when I generate: _Expected all tensors to be on the same device, but found at least two devices, cuda:1 and cuda:0! (when checking argument for argument index in method wrapper__index_select)_

I was told to run webui.py with the argument --gpu 1 (in anaconda:   python scripts/webui.py --gpu 1
In relauncher.py: additional_arguments = “–gpu 1 --esrgan-gpu 1 --gfpgan-gpu 1”  (or just --gpu 1)
In the pipeline: pipe = pipe.to("cuda:1")

They all load the diffuser on my gpu 1 (see it in task manager performance) and all give me the above error.

It loads on gpu 1, so.... what's missing to resolve the Expected tensor error?
